main/src/library/Core/System/Environment.flix

```diff
@@ -0,0 +1,61 @@
+namespace Core/System/Environment {
+  pub def getEnv(): Map[String, String] & Impure =
```

Comment:
Can we add some documentation (to all functions)?

Reply:
To make this function pure, you can wrap the whole thing (or the effectful parts) with `{ ... } as & Pure`

Reply:
Thanks